projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_bottom-nav.scss

```diff
@@ -36,14 +36,25 @@ $_dark-bootstrap-bottom-nav: extend($_bootstrap-bottom-nav, $_base-dark-bottom-n
 
 /// Generates a dark indigo bottom-nav schema based on a mix of $_indigo-bottom-nav and $_base-dark-bottom-nav
 /// @type {Map}
-/// @prop {Map} active-item-color [#fff] - The text-color used for the active color.
+/// @prop {Map} background [igx-color: 'surface'] - The background color used for the bottom-nav.
+/// @prop {Color} active-item-color [#fff] - The text-color used for the active color.
+/// @prop {Map} idle-item-color [igx-color: #fff, rgba: .6] - The text-color used for the idle color.
 /// @requires {function} extend
 /// @requires $_indigo-bottom-nav
 /// @requires $_base-dark-bottom-nav
 $_dark-indigo-bottom-nav: extend(
     $_indigo-bottom-nav,
     $_base-dark-bottom-nav,
     (
+        background: (
+            igx-color: 'surface',
+        ),
+
         active-item-color: #fff,
```

Comment:
I think this should be the contrast color to the surface color, as the palette could be anything, including a lighter surface for the dark theme.